src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), random.nextInt(101));
```

Comment:
let's avoid using magic numbers. 

see:
All magic numbers in your code should be constants.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery